src/cal-heatmap.js

```diff
@@ -1868,7 +1871,7 @@ CalHeatMap.prototype = {
 		if (this.options.highlight.length > 0) {
 			for (var i in this.options.highlight) {
 				if (this.options.highlight[i] instanceof Date && this.dateIsEqual(this.options.highlight[i], d)) {
-					return " highlight" + (this.isNow(this.options.highlight[i]) ? " now": "");
+          return this.isNow(this.options.highlight[i]) ? " highlight-now": " highlight";
```

Comment:
Since the default CSS has both, the user will still have to override the `highlight-now` class to remove the styles.  So either way they will have to include it unless they are using the default styling for it.  Correct me if I am wrong on this.  So for example, the default is 

```css
.cal-heatmap-container rect.highlight
{
	stroke:#444;
	stroke-width:1
}
```

The `highlight-now` is added with the following default

```css
.cal-heatmap-container rect.highlight-now
{
	stroke: red
}
```

Therefore if the user wants to use the `highlight` only they are still going to have to override the styling of `highlight-now`.  The only real way to save anything here is to use adjoining classes (`highlight.now`) which the linter does not allow.  Am I thinking correctly here?